elasticsearch_dsl/document_base.py

```diff
@@ -372,7 +377,7 @@ def __repr__(self):
             ),
         )
 
-    def to_dict(self, include_meta=False, skip_empty=True):
+    def to_dict(self, include_meta: bool = False, skip_empty: bool = True) -> Dict[str, JSONType]:  # type: ignore[override]
```

Comment:
I have a bunch of methods that change the return type in subclasses, and mypy does not like them. I'm ignoring these overrides one by one for now, because it seems useful to have mypy detect overrides that may be unintended.